flutter_vlc_player/android/src/main/java/software/solid/fluttervlcplayer/FlutterVlcPlayer.java

```diff
@@ -647,4 +659,231 @@ private void log(String message) {
         }
     }
 
+    // --- VLCTextureView
+
+    public class VLCTextureView extends TextureView implements TextureView.SurfaceTextureListener, View.OnLayoutChangeListener, IVLCVout.OnNewVideoLayoutListener {
```

Comment:
Do you mind moving it to a separate file?

Reply:
Yes, it's reasonable.